docs/docs/Get-Started/get-started-installation.md

```diff
@@ -3,7 +3,10 @@ title: Install Langflow
 slug: /get-started-installation
 ---
 
-You can deploy Langflow either locally or as a hosted service with [**Datastax Langflow**](#datastax-langflow).
+Langflow can be installed in three ways:
+* As a [Python package](#install-langflow-locally)
+* As a [standalone desktop application](#install-langflow-desktop)
+* As a [cloud hosted service](#datastax-langflow)
```

Comment:
Rodrigo's Slack message seems to say that he wants DS LF to be removed and LF desktop to be first.